### PR TITLE
Toggle intercom button selectively in mobile mode

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: assets/js/actions/display.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
 export const UPDATE_DISPLAY_SETTING = 'UPDATE_DISPLAY_SETTING';
 
 export const updateDisplay = (desktopOnly) => {
+  if (desktopOnly) {
+    window.Intercom('update', { "hide_default_launcher": false });
+  } else {
+    if (window.innerWidth < 721) {
+      window.Intercom('update', { "hide_default_launcher": true });
+    }
```
Comment (ambuznego) [inline assets/js/actions/display.js:9]: would there ever be a non-desktop instance where innerWidth > 721 that we would still want to hide?

Comment (vicmgs) [reply assets/js/actions/display.js:9]: not for now, anything over 721 will always go to desktop mode

Comment (ambuznego) [reply assets/js/actions/display.js:9]: do we still need the if then?

Comment (vicmgs) [reply assets/js/actions/display.js:9]: its an edge case of:

Your browser window is small.
You switch to desktop only.
You make the screen size > 721.
You click the mobile mode button but your screen size still > 721.

**Review by ambuznego — APPROVED** [review]
sendit

**Comment by vicmgs** [comment]
much thanks for monstrous catch

**Pull request merged**